pkg/cmd/uninstall.go

```diff
@@ -0,0 +1,224 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/apache/camel-k/pkg/util/kubernetes/customclient"
+
+	"github.com/apache/camel-k/pkg/client"
+	"github.com/spf13/cobra"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func newCmdUninstall(rootCmdOptions *RootCmdOptions) *cobra.Command {
+	options := uninstallCmdOptions{
+		RootCmdOptions: rootCmdOptions,
+	}
+
+	cmd := cobra.Command{
+		Use:   "uninstall",
+		Short: "Uninstall Camel K from a Kubernetes cluster",
+		Long:  `Uninstalls Camel K from a Kubernetes or OpenShift cluster.`,
+		Run:   options.uninstall,
+	}
+
+	cmd.Flags().BoolVar(&options.skipOperator, "skip-operator", false, "Do not uninstall the Camel-K Operator in the current namespace")
+	cmd.Flags().BoolVar(&options.skipCrd, "skip-crd", false, "Do not uninstall the Camel-k Custom Resource Definitions (CRD) in the current namespace")
+	cmd.Flags().BoolVar(&options.skipRoleBindings, "skip-role-bindings", false, "Do not uninstall the Camel-K Role Bindings in the current namespace")
+	cmd.Flags().BoolVar(&options.skipRoles, "skip-roles", false, "Do not uninstall the Camel-K Roles in the current namespace")
+	cmd.Flags().BoolVar(&options.skipClusterRoles, "skip-cluster-roles", false, "Do not uninstall the Camel-K Cluster Roles in the current namespace")
+	cmd.Flags().BoolVar(&options.skipIntegrationPlatform, "skip-Integration-Platform", false, "Do not uninstall the Camel-K Integration Platform in the current namespace")
```

Comment:
I think flag names are case sensitive, so you'll want `skip-integration-platform` (all lower case).

Reply:
this is right, I will fix cases